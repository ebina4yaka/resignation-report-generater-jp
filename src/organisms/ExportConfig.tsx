import { ReactElement } from 'react'
import { Container } from '@material-ui/core'
import { createStyles, makeStyles, Theme } from '@material-ui/core/styles'
import ParametersField from '../molecules/ParametersField'

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    root: {
      marginTop: theme.spacing(2),
      marginBottom: theme.spacing(2),
    },
  })
)

export default function ExportConfig(): ReactElement {
  const classes = useStyles()
  return (
    <Container className={classes.root} maxWidth="sm">
      <ParametersField />
    </Container>
  )
}
