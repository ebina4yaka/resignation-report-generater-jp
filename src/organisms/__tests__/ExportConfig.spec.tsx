import React from 'react'
import renderer from 'react-test-renderer'
import ExportConfig from '../ExportConfig'

jest.mock('../../molecules/ParametersField', () => 'MockedParametersField')

test('Snapshot test for ExportConfig', () => {
  const tree = renderer.create(<ExportConfig />).toJSON()
  expect(tree).toMatchSnapshot()
})
