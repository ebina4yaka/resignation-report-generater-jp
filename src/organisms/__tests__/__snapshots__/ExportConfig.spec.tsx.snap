// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for ExportConfig 1`] = `
<div
  className="MuiContainer-root makeStyles-root-1 MuiContainer-maxWidthSm"
>
  <MockedParametersField />
</div>
`;
